apps/site/assets/ts/schedule/components/line-diagram/SingleStop.tsx

```diff
@@ -143,12 +143,19 @@ const StopBranchLabel = (stop: RouteStop): JSX.Element | null =>
   ) : null;
 
 const StopGraphic = (isOrigin = false, isTerminus = false): JSX.Element => {
-  let yPosition = "-32px";
-  if (isTerminus) {
-    yPosition = isOrigin ? "-66px" : "32px";
+  // this hardcoded position will be shown in IE, otherwise overwritten by CSS
+  let yPosition = "32px";
+  if (isTerminus && isOrigin) {
```

Comment:
Could a stop ever be an origin without being a terminus?

Reply:
Origin might be a bit of a misnomer, as it's just the very first stop in the diagram. So, a stop could be a terminus without being an "origin".